deploy/olm/storageos/storageos.clusterserviceversion.yaml

```diff
@@ -318,14 +321,90 @@ spec:
+          cluster member IP addresses.  The node that owns the first IP address
+          listed will be responsible for bootsrapping the cluster.
+        displayName: Cluster members
+        path: join
+      - description: KV store configuration to use. Defaults to embedded. `etcd`
+          is recommended for production deployments with the address set to an
+          external etcd instance.
+        displayName: KV Store
+        path: kvBackend        
+      - description: Describes the Container Storage Interface (CSI) configuration.
+        displayName: Enable CSI
+        path: csi
+      - description: The cluster Service configuration.
+        displayName: Service configuration
+        path: service
+      - description: The shared directory where storage devices should be
+          created.  This directory must be available to both the StorageOS Node
+          container and the kubelet, and must have mount propagation enabled.
+          When kubelet is running in a container,
+          `/var/lib/kubelet/plugins/kubernetes.io~storageos` should normally be
+          set, otherwise leave empty.
+        displayName: Device directory
+        path: sharedDir
+      - description: Describes the ingress configuration to be configured for
+          the cluster.
+        displayName: Ingress configuration
+        path: ingress
+      - description: The name of the secret object that contains the etcd TLS
+          certificates.
+        displayName: etcd TLS Secret Name
+        path: tlsEtcdSecretRefName
+      - description: The namespace of the secret object that contains the etcd
+          TLS certificates.
+        displayName: etcd TLS Secret Namespace
+        path: tlsEtcdSecretRefNamespace
+      - description: Node selector terms can be set to control the placement of
+          StorageOS pods using node affiinity.
+        displayName: Node Selectors
+        path: nodeSelectorTerms
+      - description: Tolerations can be set to control the placement of
+          StorageOS pods.
+        displayName: Tolerations
+        path: tolerations                      
+      - description: To disable anonymous usage reporting across the cluster,
+          set to true. Defaults to false. To help improve the product, data such
+          as API usage and StorageOS configuration information is collected.
+        displayName: Disable Telemetry
+        path: disableTelemetry
+      - description:
+        displayName:
+        path: disableFencing
+      - description: When Pod Fencing is disabled, StorageOS will not perform
+          any interaction with Kubernetes when it detects that a node has gone
+          offline. Additionally, the Kubernetes permissions required for Fencing
+          will not be added to the StorageOS role.
+        displayName: Disable Fencing
+        path: disableFencing
+      - description: When enabled, the Operator will not perform any actions.
```

Comment:
Will not perform any action on the cluster. It's a cluster specific thingy. But since we allow only a single cluster to run at a time, it's the main cluster.